uber/utils.py

```diff
@@ -516,3 +516,15 @@ def __exit__(self, type, value, traceback):
     @staticmethod
     def _clear_cache():
         threadlocal.clear()
+
+def normalize_email(address):
```

Comment:
It also automatically lower-cases all email addresses from all domains, so I think ``normalize_email`` is a good name.  We may also apply other normalizations in the future, e.g. other providers might start ignoring dots and then we could make that configurable.